DayClock.cpp

```diff
@@ -33,7 +33,7 @@ unsigned long rollover = 0;
 
 strDateTime dateTime;
 Servo myservo;
-uint8_t today = 1;
+uint8_t today = 7;
```

Comment:
OH!  Because of the start-up dance (which I added at the last minute.).   LOL, right!   Good catch!